apps/golden-sample-app/src/app/error-page/error-page.component.ts

```diff
@@ -1,11 +1,13 @@
 import { HttpErrorResponse } from '@angular/common/http';
 import { Component, inject } from '@angular/core';
+import { CommonModule } from '@angular/common';
 import { Router } from '@angular/router';
+import { ErrorCommonStateModule } from '@backbase/ui-ang/common-error-state';
 
 @Component({
   selector: 'app-error-page',
   templateUrl: './error-page.component.html',
-  standalone: false,
+  imports: [CommonModule, ErrorCommonStateModule],
```

Comment:
I don't think we need to import Angular's `CommonModule` in this component - I don't see any uses of anything from it in the [template](https://github.com/Backbase/golden-sample-app/blob/e8c0246e015025225666dbfa0f9248593a2f7fdf/apps/golden-sample-app/src/app/error-page/error-page.component.html)